cfme/tests/automate/test_domain.py

```diff
@@ -90,3 +90,65 @@ def test_cannot_edit_builtin():
         assert not details_view.configuration.item_enabled('Edit this Domain')
     else:
         assert not details_view.configuration.is_displayed
+
+
+@pytest.mark.tier(2)
+def test_domain_name_wrong():
+    domains = DomainCollection()
+    with error.expected('Name may contain only'):
+        domains.create(name='with space')
+
+
+@pytest.mark.tier(2)
+def test_domain_lock_unlock(request):
+    domains = DomainCollection()
+    domain = domains.create(
+        name=fauxfactory.gen_alpha(),
+        description=fauxfactory.gen_alpha(),
+        enabled=True)
+    request.addfinalizer(domain.delete)
```

Comment:
The delete failed against 5.6, Could not find an element u'./ul/li/a[normalize-space(.)="Remove this Domain"]' - might need to version pick the configuration dropdown item.

Its outside your narrow scope here, up to you to fix it here or just write issue.